addons/stock_account/models/product_value.py

```diff
@@ -56,7 +56,7 @@ def _compute_current_value_details(self):
             move = product_value.move_id
             quantity = move.quantity
             uom = move.product_uom.name
-            price_unit = move.value / move.quantity
+            price_unit = move.value / move.quantity if move.quantity else 0.0
```

Comment:
I think it should be better to skip the computation in this case since the computed text has no use and we already skip the computation if there is no product's move (so you can simply adapt the condition)